JSON-connect/src/appsscript.json

```diff
@@ -1,11 +1,11 @@
 {
   "dataStudio": {
-    "name": "JSON-connect",
-    "logoUrl": "https://github.com/Lingewoud/JSON-community-connector/raw/master/assets/logo.png",
+    "name": "JSON Connect",
+    "logoUrl": "https://raw.githubusercontent.com/googledatastudio/community-connectors/master/JSON-connect/assets/logo.png",
     "company": "Lingewoud",
     "companyUrl": "https://lingewoud.nl/",
     "addonUrl": "https://github.com/Lingewoud/JSON-community-connector",
```

Comment:
'Learn more' (addonUrl) links should point to repo from which code is deployed, i.e. this repo